index.html

```diff
@@ -144,13 +144,15 @@ <h4 class="aboutme__content__h4">About me</h4>
 
 <div class="form__content" id="contact">
     <p class="form_p">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-    <form class="form" action="https://formspree.io/f/mayknebl" method="post">
-        <input type="text" name="fullname" placeholder="Full name" maxlength="30" required />
-        <input type="email" name="emailaddress" placeholder="Email address"  required />
+    <form id="form" class="form" action="https://formspree.io/f/mayknebl" method="post">
+        <input type="text" name="fullname" id="fullname" placeholder="Full name" maxlength="30" required />
+        <input type="email" name="emailaddress" id="emailaddress" placeholder="Email address"  required />
 
-        <textarea  name="txt_area" class="textarea" placeholder="Enter text here" maxlength="500" required></textarea>
-        <button class="form_btn" type="submit">Get in Touch</button>
+        <textarea  name="txt_area" class="textarea" id="textareaForm" placeholder="Enter text here" maxlength="500" required></textarea>
+        <button class="form_btn" id="form_btn" type="submit">Get in Touch</button>
+        <div class="msj_alert hidden"></div>
     </form>
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I  think it would be better if you display your **alert message** between the `textarea field` and the `Get In Touch`. This will help the user if he didn't **scroll** the entire page(up to the bottom) to **see the error message**. But Good Job :clap: :clap:

- [ **OPTIONAL** ] I also suggest you use the `red` color to display the **alert message** as the color most used to display error messages.